npm-debug.log

```diff
@@ -0,0 +1,19 @@
+0 info it worked if it ends with ok
```

Comment:
Leftover ?

Reply:
I don't know what it is. So I will remove it